USERGUIDE.md

```diff
@@ -221,153 +222,210 @@ extension ViewController: MiniAppMessageDelegate {
 }
 ```
 
-<div id="navigation"></div>
+<a id="user-profile-details"></a>
 
-### Add a web navigation interface to the MiniApp view
+##### Retrieve User Profile details
 ---
-MiniApp iOS SDK provides a fully customizable way to implement a navigation interface inside your html pages with a `MiniAppNavigationConfig` object. The class takes 3 arguments:
+**API Docs:** [MiniAppUserInfoDelegate](https://rakutentech.github.io/ios-miniapp/Protocols/MiniAppUserInfoDelegate.html)
 
-- `navigationBarVisibility` : 
-    - never = the UI will never be shown
-    - auto = navigation UI is only shown when a back or forward action is available
-    - always = navigation UI is always present
-- `navigationDelegate` : A delegate that will receive MiniApp view instructions about available navigation options. It will also receive taps on external links.
-- `customNavigationView` : A view implementing `MiniAppNavigationDelegate` that will be overlayed to the bottom of the MiniApp view
+Get the User profile related details using 'MiniAppMessageDelegate'.
+The following delegates/interfaces will be called only if the user has allowed respective [Custom permissions](#custom-permissions)
 
-```swift
-let navConfig = MiniAppNavigationConfig(
-                    navigationBarVisibility: .always,
-                    navigationDelegate: myNavigationDelegate,
-                    customNavigationView: mCustomView)
+<a id="user-profile-details-username"></a>
 
-MiniApp.shared(with: Config.getCurrent(), navigationSettings: navConfig).info(miniAppId: miniAppID) { (result) in
-...
+#### User Name
```

Comment:
I think this should be `######` so it's smaller than `Retrieve User Profile details`:

```suggestion
###### User Name
```